### ISSUE TITLE
Icon requests

### ISSUE THREAD

**Issue by Spinnekop** [EliverLara/candy-icons#84]
OpenLP
Distributor logo - Arch
Plasma menu - internet, office, graphics, system
pamac-manager-install (to link to software)
hplip
latte
brasero

Great icon set!!

**Comment by Katana97** [comment]
Yes! Thanks for the great theme, but can we get 'No 3' above, the menu icons for internet, office, graphics, system. I notice they don't match every time I open the menu.

**Comment by WhiteBahamut** [comment]
would be great to get No 3. But actually "office" has an icon on my menu, but "Development" and "education" not

![image](https://user-images.githubusercontent.com/38832027/74920956-4a68ba80-53cd-11ea-846f-6a42369ae92d.png)
